infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/fsm/InitializedState.java

```diff
@@ -48,6 +51,7 @@ public LogReplicationState processEvent(LogReplicationEvent event) throws Illega
                 return waitSnapshotApplyState;
             case LOG_ENTRY_SYNC_REQUEST:
                 log.info("Start Log Entry Sync, requestId={}", event.getEventId());
+                fsm.getAckReader().setSyncType(SyncType.LOG_ENTRY);
```

Comment:
this already gets set on entry to LogEntrySyncState

Reply:
I was thinking about this as well, same for the above events. Setting the syncType here isn't needed, however the reason for this PR stemmed from a bug where the polling task would display inaccurate info since the correct info hadn't been populated yet by the entry to whichever state. I think setting the information as soon as we have it available might help avoid ambiguity in the future even though we do not have that problem with these changes.

Reply:
since we havent entered the state yet, we should not update the sync type here.